### ISSUE TITLE
Model sharing

### ISSUE THREAD

**Issue by thomasnguyen92** [leiboliu/xr-lat#1]
Hi Leibo,
First and foremost, I want to express my gratitude for your valuable contribution to the field through your paper, "Automated ICD Coding Using Extreme Multi-Label Long Text Transformer-Based Models." It has significantly broadened my understanding of the state-of-the-art results in ICD coding, and I am truly appreciative of the work you’ve done.

Having delved into your paper, I am particularly interested in the BL-5 model and the impressive results showcased in Table 2. I believe that studying the model and its settings in depth could provide critical insights and advance my own work in this domain.

With that in mind, I was wondering if you would be open to sharing the BL-5 model along with its inference settings that led to the best results in Table 2. Access to this information would be instrumental in helping me and possibly others in the community to better understand the practical applications of your findings.

**Comment by leiboliu** [comment]
Thanks Thomas. I am happy to share BL-5 model. Please download it [here](https://unsw-my.sharepoint.com/:f:/g/personal/z5250377_ad_unsw_edu_au/EiOITvMgOAhMgZrqsCkIsakBNObT_gKoxGt0fMv4X7jU9g). The easy way to only predict on test set is to add the following code into the line 477 of xr-lat/run_coding.py and use config-baseline.json as config file. Please remember to set do_train and do_eval as false in the config file.

`
best_model = "" #BL-5 model path
state_dict = torch.load(best_model, map_location=training_args.device)
hilat_model.load_state_dict(state_dict, strict=False)
trainer.model = hilat_model

`
